frontend/src/metabase/admin/datamodel/containers/FieldApp.jsx

```diff
@@ -310,8 +310,7 @@ const FieldGeneralPane = ({
           <Select
             className="inline-block"
             value={
-              field.json_unfolding ||
-              (database.details["json-unfolding"] ?? true)
+              field.json_unfolding ?? database.details["json-unfolding"] ?? true
```

Comment:
This is in the [unfold-json-by-field](https://github.com/metabase/metabase/tree/unfold-json-by-field) branch, so I'm not sure why it's appearing here in the github UI